tests/test_routes.py

```diff
@@ -10,8 +10,10 @@
 from unittest import TestCase
 from unittest.mock import MagicMock, patch
 from service import app
-from service.models import db
+from service.models import Promotion, DataValidationError, db
 from service.common import status  # HTTP Status Codes
+from service.helpers import convert_data, convert_data_back
+from tests.factories import PromoFactory
 
```

Comment:
why don't we have a “_create_promotions” function here?

Reply:
It would be great if you can add some logging.debug in tests?

Reply:
_create_promotions function will create promotions in bulk which would be helpful and convenient for reading and listing promotions.

Reply:
Its a good idea, I am open to writing it for a future use case. I am adding an issue in the Icebox in Zenhub for us to make this in the future.